components/nimbus/src/enrollment.rs

```diff
@@ -496,6 +517,19 @@ pub enum EnrollmentStatus {
     },
 }
 
+impl EnrollmentStatus {
+    pub fn name(&self) -> String {
+        match self {
+            EnrollmentStatus::Enrolled { .. } => "Enrolled",
+            EnrollmentStatus::NotEnrolled { .. } => "NotEnrolled",
+            EnrollmentStatus::Disqualified { .. } => "Disqualified",
+            EnrollmentStatus::WasEnrolled { .. } => "WasEnrolled",
+            EnrollmentStatus::Error { .. } => "Error",
+        }
+        .into()
+    }
+}
```

Comment:
So I couldn't use the `fmt::Debug` thing here because it ends up including the values associated with each enum and that 
1. makes testing difficult at best
2. shouldn't be sent to Glean according to the metric spec

If there's a better way I could've done this please let me know!